README.md

```diff
@@ -0,0 +1,223 @@
+
+<!-- *[Bookstore]* can do math calculation like add, subtract, multiply and division. -->
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+> I use HTML/js  REACT for the front-end.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+
+
+### Key Features <a name="key-features"></a>
+
+<!-- > Describe between 1-3 key features of the application. -->
+
+- *[JS OBJECTS]*
+- *[BASIC HTML]*
+  <p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+
+
+
+
+
+<!-- - [Live Demo Link]() -->
+[demo](https://develop--zesty-mooncake-1a3672.netlify.app/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+- Code Editor
+- Git installed
+- Node js installed
+
+You need to have Node.js installed.
+
+### Setup
+
+
+Clone this repo into your desire folder and then cd into it.
+https://github.com/amklass11/react-capsyone.git
+
+### Install
+
+sh
+npm install
+
+
+### Usage
+
+Run following command to start development server with live update.
+
+sh
+npm run start
+
+
+To build the app
+
+sh
+npm run build
+
+
+### Run tests
+
+There is no tests in this project but if you want to check linter errors run following command.
+
+sh
+npm run linters
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Amk Lass**
+
+
+- GitHub:[@amklass11](https://github.com/amklass11)
+
+- LinkedIn:[@AmkLass](https://www.linkedin.com/in/amk-lass-521565196/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+<!-- > Describe 1 - 3 features you will add to the project. -->
+
+- [ ] *[Add drag and drop function]*
+- [ ] *[Update UI/UX Design]*
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please don't forget to follow me and give me advice so that I can improve more.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Kindly acknowledge [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).